assets/img/reporting-mode-basic-architecture.svg

```diff
@@ -0,0 +1 @@
5 -0.109375 0.734375 -0.109375zm5.6759033 5.203125q0.21875 0 0.4375 -0.03125q0.21875 -0.03125 0.359375 -0.09375l0 0.671875q-0.140625 0.078125 -0.421875 0.125q-0.265625 0.046875 -0.515625 0.046875q-0.453125 0 -0.828125 -0.15625q-0.359375 -0.171875 -0.578125 -0.546875q-0.21875 -0.375 -0.21875 -1.046875l0 -3.390625l-0.8125 0l0 -0.421875l0.8125 -0.328125l0.34375 -1.25l0.515625 0l0 1.328125l1.671875 0l0 0.671875l-1.671875 0l0 3.359375q0 0.53125 0.25 0.796875q0.25 0.265625 0.65625 0.265625z" fill-rule="nonzero"/><path fill="#000000" fill-opacity="0.0" d="m214.50919 130.01312l-2.6955872 0l0 0.724411l-89.17843 0" fill-rule="evenodd"/><path stroke="#00aae4" stroke-width="2.0" stroke-linejoin="round" stroke-linecap="butt" d="m202.53107 130.73753l-67.89589 0" fill-rule="evenodd"/><path fill="#00aae4" stroke="#00aae4" stroke-width="2.0" stroke-linecap="butt" d="m202.7305 134.03497l8.860214 -3.8453522l-9.259064 -2.749527z" fill-rule="evenodd"/><path fill="#00aae4" stroke="#00aae4" stroke-width="2.0" stroke-linecap="butt" d="m134.63518 127.43407l-9.076202 3.3034668l9.076202 3.3034668z" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m122.509186 137.01575l89.00787 0l0 18.960632l-89.00787 0z" fill-rule="evenodd"/><path fill="#000000" d="m160.25713 139.64638q0.9375 0 1.546875 0.234375q0.609375 0.234375 0.90625 0.703125q0.3125 0.46875 0.3125 1.203125q0 0.609375 -0.21875 1.015625q-0.21875 0.40625 -0.578125 0.65625q-0.34375 0.25 -0.75 0.375l2.09375 3.421875l-1.03125 0l-1.875 -3.171875l-1.578125 0l0 3.171875l-0.875 0l0 -7.609375l2.046875 0zm-0.046875 0.765625l-1.125 0l0 2.921875l1.203125 0q0.9375 0 1.375 -0.375q0.4375 -0.390625 0.4375 -1.125q0 -0.78125 -0.46875 -1.09375q-0.453125 -0.328125 -1.421875 -0.328125zm6.5612793 -0.765625q1.46875 0 2.140625 0.578125q0.6875 0.578125 0.6875 1.640625q0 0.484375 -0.15625 0.921875q-0.15625 0.4375 -0.515625 0.765625q-0.359375 0.328125 -0.9375 0.53125q-0.5625 0.1875 -1.40625 0.1875l-0.921875 0l0 2.984375l-0.875 0l0 -7.609375l1.984375 0zm-0.09375 0.75l-1.015625 0l0 3.109375l0.828125 0q0.734375 0 1.21875 -0.15625q0.484375 -0.15625 0.71875 -0.5q0.25 -0.359375 0.25 -0.9375q0 -0.765625 -0.484375 -1.140625q-0.484375 -0.375 -1.515625 -0.375zm7.7697754 -0.078125q-0.640625 0 -1.140625 0.21875q-0.5 0.21875 -0.859375 0.625q-0.34375 0.40625 -0.53125 0.984375q-0.1875 0.578125 -0.1875 1.296875q0 0.96875 0.296875 1.671875q0.3125 0.703125 0.90625 1.09375q0.59375 0.375 1.484375 0.375q0.515625 0 0.953125 -0.078125q0.453125 -0.09375 0.890625 -0.21875l0 0.765625q-0.421875 0.15625 -0.890625 0.234375q-0.453125 0.078125 -1.078125 0.078125q-1.15625 0 -1.9375 -0.484375q-0.78125 -0.484375 -1.171875 -1.359375q-0.375 -0.890625 -0.375 -2.078125q0 -0.859375 0.234375 -1.578125q0.25 -0.71875 0.703125 -1.234375q0.46875 -0.53125 1.140625 -0.8125q0.6875 -0.28125 1.5625 -0.28125q0.578125 0 1.109375 0.125q0.546875 0.109375 0.984375 0.3125l-0.359375 0.75q-0.359375 -0.15625 -0.796875 -0.28125q-0.4375 -0.125 -0.9375 -0.125z" fill-rule="nonzero"/></g></svg>
```

Comment:
The `assets/` folder is for files associated with the templates. Move this in the top-level `img/` folder, which is for files associated with the contents.

That's one of the things causing this to fail to build at the moment.